internal/pkg/network/setup.go

```diff
@@ -92,7 +92,7 @@ func defaultNetworkSetup() (err error) {
 	return nil
 }
 
-func ifup(ifname string) (err error) {
+func ifup(ifname string, mtu int) (err error) {
```

Comment:
should we consider option interface (or refactoring interface to be more of an object?) to support more parameters in the future, and also avoid passing `0` as default value?

Reply:
Yeah, this is a bit of a stopgap for now. This will likely get rewritten as part of the networkd rewrite that Brad is working on.

Reply:
Yeah, there's a significant refactor to this network code that needs to happen and we just haven't gotten around to yet. I think for now this should be fine.